### DISCUSSION TITLE
Can you omit the virtual side of a relationship?

### DISCUSSION THREAD

**Discussion by Erhannis** [collerek/ormar#106] [Q&A] [answered]
Oftentimes, I only care about a relationship in one direction, and don't need the mirror field on the far object.  In fact, it can be kinda problematic, as it can fill up e.g. `.json()` with junk about a parent that's already been printed, and I have to either ignore it, or figure out how to selectively remove that information.

1. Is there a parameter to disable the automatic creation of the far-side field representing a relationship?
2. If not, would it _break_ something for such a parameter to exist?  Like, does ormar internally require both fields to be present in order to do such-and-such bookkeeping, and would be very hard to rewrite in a way that didn't require it?
3. (If not, this will turn into a feature request, haha.)

**Comment by collerek** [comment]
Right now there is no such parameter, i can check if this will be easy to implement, probably it would take a form of parameter in `ForeingKey` and `ManyToMany` like `skip_reverse` flag so you can control this per relation.

**Reply by Erhannis** [reply]
Thanks; lemme know what you find.

**Comment by collerek** [comment, marked as answer]
This was added in 0.10.3
